### ISSUE TITLE
Deleted

### ISSUE THREAD

**Issue by PercyTheB** [MicrocontrollersDev/Lunar-Client-Alternatives#7]
Hello your repository https://github.com/MicrocontrollersDev/Alternatives/blob/main/LunarClient.md got deleted! just informing you. Maybe you can make a discord/pastebin with the details?

**Comment by MicrocontrollersDev** [comment]
Yeah, I currently have a GitHub support ticket open to try and get it back
up and running. In the meantime, I reopened my old alternatives lists and
the lunar client one can temporarily be found here:
https://github.com/MicrocontrollersDev/Lunar-Client-Alternatives
